base/uk.ac.stfc.isis.ibex.opis/resources/hrpd_in_situ_dsc.opi

```diff
@@ -0,0 +1,1651 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this needs to be added into `opi_info.xml` as it currently doesnt show up in the device screens list

Reply:
Good spot, added now